Bottom: 04cbb05f65709eca7cb7613e90eb1751cea33548
Top:    466ed2ce48fce496644194268b2087a30dc54834
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 17:51:50 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 358d8c8..f1384d7 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -298,8 +298,8 @@ ghid_use_mask (int use_it)
   switch (use_it)
     {
     case HID_MASK_BEFORE:
-      /* Ignore, this renderer doesn't use it */
-      return;
+      /* The HID asks not to receive this mask type, so warn if we get it */
+      g_return_if_reached ();
 
     case HID_MASK_CLEAR:
       /* Write '1' to the stencil buffer where the solder-mask should not be drawn. */
